mods/tuxemon/db/locale/en_US.json

```diff
@@ -80,6 +80,13 @@
     "menu_player": "Player",
     "menu_run": "Run",
     "menu_save": "Save",
+    "menu_up_key": "Up Key",
+    "menu_left_key": "Left Key",
+    "menu_right_key": "Right Key",
+    "menu_down_key": "Down Key",
+    "menu_primary_select_key": "P-Select Key",
```

Comment:
I've been using "Action" and "Cancel" for the button names.  Does that seem ok?  I'm not sure is "P-Select" and "S-Select" are clear enough.

Reply:
Oh, I didn't know the names that were used in the actual code. I'll rename them now.